tensorflow/core/api_def/base_api/api_def_WindowDataset.pbtxt

```diff
@@ -4,29 +4,82 @@ op {
   in_arg {
     name: "size"
     description: <<END
-A scalar representing the number of elements to accumulate in a window.
+A `tf.int64` scalar `tf.Tensor`, representing the number of elements
+of the input dataset to combine into a window. Must be positive.
 END
   }
   in_arg {
     name: "shift"
     description: <<END
-A scalar representing the steps moving the sliding window forward in one
-iteration. It must be positive.
+A `tf.int64` scalar `tf.Tensor`, representing the number of input elements
+by which the window moves in each iteration.  Defaults to `size`.
+Must be positive.
 END
   }
   in_arg {
     name: "stride"
     description: <<END
-A scalar representing the stride of the input elements of the sliding window.
-It must be positive.
+A `tf.int64` scalar `tf.Tensor`, representing the stride of the input elements
+in the sliding window. Must be positive. The default value of 1 means
+"retain every input element".
 END
   }
   in_arg {
     name: "drop_remainder"
     description: <<END
-A scalar representing whether a window should be dropped in case its size is
-smaller than desired.
+A `tf.bool` scalar `tf.Tensor`, representing whether the last window should be
+dropped if its size is smaller than `window_size`.
 END
   }
-  summary: "A dataset that creates window datasets from the input dataset."
+  summary: <<END
+  C++ implementation of the Python API `tf.data.Dataset.window()`.
```

Comment:
Remove this line, the base api is meant to be language agnostic.

Reply:
Removed this line in commit 84bc9b4